xbmc/input/mouse/MouseStat.h

```diff
@@ -53,6 +53,14 @@ enum MOUSE_BUTTON
   MOUSE_EXTRA_BUTTON4
 };
 
+enum class HoldAction
+{
+  NONE, //no action should occur
```

Comment:
Unsure this is recognized by doxygen. It would be nice if you could check.
I think you need to move the comment above the enum value and add a space after each //

Reply:
i have fixed all doxy in separate commit